### DISCUSSION TITLE
Accessing authelia on local network

### DISCUSSION THREAD

**Discussion by Cybertom3000** [authelia/authelia#7759] [Q&A] [answered]
Hello,
I've just installed authelia on my Unraid and get an error message when I try to access the WebUI.
"Error occurred retrieving user session" error="unable to retrieve session cookie domain provider: no configured session cookie domain matches the url '[http://192.168.xxx.6:9091/api/state'](javascript:void(0);)" method=GET path=/api/state remote_ip=192.168.xxx.4

THX in advance

[configuration.txt](https://github.com/user-attachments/files/16749754/configuration.txt)

**Comment by james-d-elliott** [comment]
Authelia doesn't support client connections via insecure schemes for starters. You will need to access this via your proxy and configure the [required headers](https://www.authelia.com/integration/proxies/introduction/#required-headers). In addition every hostname you access authelia via must have a valid cookie domain configured (either the same as the hostname, or the hostname is a subdomain of that domain).

**Comment by Cybertom3000** [comment]
Thanks a lot for your reply.
I've integrated Authelia into the NginX Proxy Manager and followed [the instructions](https://app.gitbook.com/o/-Mbz568o5Qo_VLAdXaS7/s/DrboGm7zMbJvDEJu9VCi/nginx/nginx-config-authelia)

I put this in ‘Advanced’ tab, set my server IP 192.168.178.6:9091 and adjusted the standard IP range of the Fritzbox 192.168.178.0/24 below.
When I call up Authelia via my domain https://auth.mydomain.icu/, I get a white page with ‘Page not found’

auth.mydomain.icu / mydomain.icu is the configured subdomain / hostname. I'm not sure, where I'm wrong.

![grafik](https://github.com/user-attachments/assets/d1631f77-0c76-4c69-bd8c-5badacecef82)

**Reply by james-d-elliott** [reply]
The configuration issue is in your proxy. What does the not found page look like and what do the proxy logs look like?

**Reply by Cybertom3000** [reply]
The files from NPM are attached. 
[proxy.txt](https://github.com/user-attachments/files/16769871/proxy.txt)

The white page looks like this 

![grafik](https://github.com/user-attachments/assets/eceda6ec-cad2-4245-9cdd-1da60c148c5b)

THX again

**Reply by james-d-elliott** [reply]
Can you translate that into English for me? Looks like a DNS error

**Reply by hendrik1120** [reply, marked as answer]
It indeed is a dns error, probably forgot to set the dns record to your reverse proxy, or you might need to clear the browser dns cache.
Had to check it myself. I have all my browsers set to English as the translation is not very accurate.

**Reply by Cybertom3000** [reply]
THX! I can't believe not to double check this. I set the dns entry but probably forgot to save it. Now the dns entry is completed

@james-d-elliott Thank you for your effort.

**Comment by Cybertom3000** [comment]
Sorry, the translation:

**Page was not found**

The connection to the server auth.mydomain.icu failed.

If you have entered the correct address, you can do the following:

- Try again later.
- Check your network connection.
- Check that Firefox is authorised to access the Internet (you may be connected but behind a firewall).
